server/api/rating/imdbRadarrProxy.ts

```diff
@@ -0,0 +1,199 @@
+import ExternalAPI from '@server/api/externalapi';
+import cacheManager from '@server/lib/cache';
+
+export type IMDBRadarrProxyResponse = IMDBMovie[];
+
+export interface IMDBMovie {
```

Comment:
Let's not export interfaces that are only used in this one file. (Same goes for other interfaces in here)